src/components/Article.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { BsArrowRight, BsArrowDown } from "react-icons/bs";
+import { Link } from "react-router-dom";
+
+export default function Article({ articles, title }) {
+  return (
+    <div className=" grid grid-cols-4 md:grid-cols-1 gap-24 items-start">
+      {/* All articles */}
+      <div className="col-span-full">
+        <h1 className="text-2xl flex items-center gap-3  font-bold tracking-wide uppercase  static top-3">
+          {title}
+          <BsArrowDown className="text-lg" />
+        </h1>
+        {articles.length > 0 ? (
```

Comment:
The length is 0 or a number greater than zero, and 0 it's falsy, so the good way in this case could be just to check for the length as boolean
```suggestion
        {articles.length ? (
```